### DISCUSSION TITLE
How to decompose LOS deformation?

### DISCUSSION THREAD

**Discussion by BelledDeng** [gmtsar/gmtsar#363] [Q&A]
Dear Eric Xu,
I want to decompose LOS deformation into  east–west motion  and up–down and south–north motion like the Figure 3c-d in your Ridgecrest article.
![image](https://user-images.githubusercontent.com/105110883/177902008-90bd6eca-abba-4222-8a26-d8c89667efe7.png)
However, I don't know how to decompose LOS deformation. Could you tell me what shall I do?
Yours,
Belled Deng

**Comment by Xiaohua-Eric-Xu** [comment]
Hi @BelledDeng, you could follow what's described in the paper.
1) get unwrapped LOS deformation from descending and ascending map.
2) make a decomposition taking advantage that the look angle are largely symmetric in the region for the two tracks.
3) high-pass filter the decomposed maps to let small details show up.
There are quite some literatures on 2) and the simplest way is to add/subtract and then divide by 2 times the look cosines.

**Reply by Congzhangg** [reply]
@Xiaohua-Eric-Xu @BelledDeng Hello ,Have you got the 3D deformation map? Can you provide some examples or tutorials?

**Reply by BelledDeng** [reply]
Hi @Congzhangg .If the 3D deformation map is the deformation in the east-west, north-south and vertical , you can try MAI_processing.csh to get the deformation in range/azimuth. Then, you will get the 3D deformation map. But more information you should ask @Xiaohua-Eric-Xu .

**Reply by Xiaohua-Eric-Xu** [reply]
The MAI_processing.csh can provide another component along the track direction. It has lower quality, but should be comparable to a crude cross-correlation result.

**Reply by KIKINEO** [reply]
@Xiaohua-Eric-Xu Dear Professor Xu, I recently want to use your MAI_processing.csh program to process the Sentinel 1Asar image to obtain the azimuthal shape of the surface, but I am a little confused about the format of the input master and slave images and the config file, could you please provide some more specific instructions, I look forward to your reply.

**Reply by Xiaohua-Eric-Xu** [reply]
MAI_processing.csh could not deal with Sentinel-1 data for the moment. The procedure to do that requires all doppler being properly sampled along azimuth. Due to the electronic steering of the antenna of Sentinel-1, there are extra dopplers introduced along azimuth and one has to properly remove that to perform MAI. We could use the demod-deramp functions in the metadata to do the remove (code in make_s1a_tops.c), but I think the function may not be accurate enough to fully remove the ramp. I know the GAMMA software does a complicated procedure to remove the ramp and we never took the effort toward implementing that, mainly because the 14m sampling along azimuth of Sentinel-1 makes MAI really insensitive.

**Reply by Congzhangg** [reply]
@Xiaohua-Eric-Xu May I ask how to decompose vertical deformations when LOS direction deformations are obtained? Is there a script to achieve this function？

**Reply by Xiaohua-Eric-Xu** [reply]
First of all, you cannot get 3D deformation with 2 LOS (descending and ascending) measurements. Overall the near-polar orbits are not very sensitive to north motion. 

To do a simple decomposition, add or subtract the two LOS measurement (des +/- asc and then divide by two) and then scale them with the vertical look cosine, you'll have east-west and vertical deformation maps.

**Reply by Alzaidy1** [reply]
Interested

**Reply by ditafaith** [reply]
If my understanding is correct, based on the explanation above from @Xiaohua-Eric-Xu, the GMTSAR currently can't use to decompose or obtain north-south components. Is it correct ? @BelledDeng

**Reply by Xiaohua-Eric-Xu** [reply]
For Sentinel-1, the MAI part does not work for now, mainly because during pre-processing, we restores the ramp (using parameters inside the metadata). This was due to that when we were developing the code, we didn't trust the ramp parameters so remove-resamp-restore strategy was used. Still, you could run xcorr to get azimuthal shift, the quality should be around the same. Or you may use spectral_diversity.c to get burst overlap interferograms too.

**Comment by ditafaith** [comment]
> Dear Eric Xu, I want to decompose LOS deformation into east–west motion and up–down and south–north motion like the Figure 3c-d in your Ridgecrest article. ![image](https://user-images.githubusercontent.com/105110883/177902008-90bd6eca-abba-4222-8a26-d8c89667efe7.png) However, I don't know how to decompose LOS deformation. Could you tell me what shall I do? Yours, Belled Deng

Hi @BelledDeng, can you share your experience on how to decompose using GMTSAR?
Currently, we have all LOS files in both ascending and descending directions. However, which scripts I should use and the procedure are not quite clear.

**Reply by Xiaohua-Eric-Xu** [reply]
There's no existing scripts to do so. I used a few GMT commands to get the grids above. 1) decompose the grid with adding and subtracting the descending and ascending grids while making a scaling based on the look vectors. 2) run a `gmt grdfilter` with a high-pass filter specified.

**Reply by ditafaith** [reply]
Thanks @Xiaohua-Eric-Xu for explaining this step.
